src/graph.rs

```diff
@@ -769,15 +769,16 @@ impl ModuleGraph {
   /// returning the "final" module.
   pub fn resolve(&self, specifier: &ModuleSpecifier) -> ModuleSpecifier {
     let mut redirected_specifier = specifier;
-    let mut seen = HashSet::new();
+    let max_redirects = 6;
```

Comment:
Does this match the number of allowed redirects in CLI file fetcher?

Reply:
That is 10, I will increase this